src/h_vialib/client.py

```diff
@@ -22,6 +24,11 @@ def __init__(self, url, content_type=None):
 
         self._content_type = content_type
 
+    @property
+    def is_html(self):
+        """Check if document is known to be a pdf."""
```

Comment:
```suggestion
        """Check if document is known to be a HTML."""
```

Reply:
It's almost as if I copy pasted this... _almost_